general/clients/codec-support.md

```diff
@@ -67,23 +67,25 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 ## [Subtitle Compatibility](https://en.wikipedia.org/wiki/Comparison_of_video_container_formats#Subtitle/caption_formats_support "Wikipedia's subtitle codec tables")
 
-Subtiles can be a subtle issue for transcoding. Containers have a limited number of subtitles that are supported. If subtitles need to be transcoded, it will happen one of two ways. They can be converted into another supported format (text-based subtitles) or burned into the video (image/lossless based and ASS based) due to the subtitles transcoding not being supported. This is the most intenstive method of transcoding due to two transcodings happening at once; applying the subtitle layer on top of the video layer.
+Subtiles can be a subtle issue for transcoding. Containers have a limited number of subtitles that are supported. If subtitles need to be transcoded, it will happen one of two ways. They can be converted into another supported format (text-based subtitles) or burned into the video (image/lossless based and ASS based) due to the subtitles transcoding not being supported. This is the most intenstive method of transcoding due to two transcodings happening at once; applying the subtitle layer on top of the video layer. [Breakdown of subtitles](https://www.afterdawn.com/guides/archive/subtitle_formats_explained.cfm).
 
-||Format|TS|MP4<sup>3</sup>|MKV|AVI|
+||Format|TS|MP4<sup>1</sup>|MKV|AVI|
 |:---:|:---:|:---:|:---:|:---:|:---:|
 |[SubRip Text (SRT)](https://en.wikipedia.org/wiki/SubRip)|Text|❌|🔶|✅|🔶|
-|[WebVTT (VTT)](https://en.wikipedia.org/wiki/WebVTT)|Text|❌<sup>1</sup>|❌<sup>1</sup>|✅|🔶|
-|ASS/SSA<sup>1</sup>|Formatted Text|❌|❌|✅|🔶|
-|VobSub|Picture|❌|✅|✅|🔶|
-|DVB-SUB [(SUB + IDX)](https://forum.videohelp.com/threads/261451-Difference-between-SUB-and-IDX-file)|Picture|✅|✅|✅|❌|
+|[WebVTT (VTT)](https://en.wikipedia.org/wiki/WebVTT)<sup>2</sup>|Text|❌|❌|✅|🔶|
+|ASS/SSA<sup>3</sup>|Formatted Text|❌|❌|✅|🔶|
+|VobSub<sup>4</sup>|Picture|✅|✅|✅|🔶|
+||Picture|✅|✅|✅|❌|
```

Comment:
Did you mean to remove this line entirely?

Reply:
For now yeah, i changed sub + idx to a footnote since it's 'supposed to be the same' as vobsub. Once I get into my custom codecs stages, I'll see how they all interoperate with each other and I might need to expand it back out.

Reply:
I would leave the title at least then, just so people know what this row does.

Reply:
Did I mess with any titles? Since Vobsub should match sub/idx compatibility, i removed it since the table is a breakdown of whether support is available or not for each type.